client/components/action-panel/style.scss

```diff
@@ -12,6 +12,13 @@
 	line-height: 24px;
 	color: var( --color-neutral-70 );
 	clear: none;
+
+	.is-placeholder & {
```

Comment:
Updating base components can be done, but it carries more risk. For ones that have a lot of usages we tend to pull updates out into its own PR. 'll leave it up to you @blackjackkent on what you'd prefer for next steps.

1. Split out a PR to add placeholder support for action-panel (what we have here), update relative docs/example.jsx with a placeholder example (this populates https://wpcalypso.wordpress.com/devdocs/design/action-panel ). Update summary with testing instruction to test for regressions all ActionPanel usages (which includes subcomponents).

2. Split out a PR  to scope placeholder support to the PromoCard component instead + update docs/example.jsx (smaller number of usages)

3. Scope placeholder support to PromoCards within the earn section in this PR. (styles would be added to `client/my-sites/earn/styles.scss`